pylammpsmpi/wrapper/concurrent.py

```
@@ -402,8 +402,36 @@ def installed_packages(self):
+                return variable
+
+        args = list(args)
+        if len(args) == 3 and args[2]:
+            if isinstance(args[2], list):
+                args[2] = [check_type(variable=a) for a in args[2]]
+            elif isinstance(args[2], dict):
+                args[2] = {k: check_type(variable=v) for k, v in args[2].items()}
+            elif isinstance(check_type(variable=args[2]), str):
+                args[2] = "pylammpsmpi.lammps.reference"
+            else:
+                raise TypeError(f"Argument type {type(args[2])} not supported")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)